### ISSUE TITLE
Mangonels throw cows

### ISSUE THREAD

**Issue by GRhin** [Sh0wdown/UnofficialCrusaderPatch#654]
Reason: Almost all current defense works are primarily anti melee. Traps are all range limited so generally only deploy on melee, ballista are primarily anti seige (too low fire rate to do too much to seige forces) and mangonels are too inaccurate at range, so dont do too much damage there. This is not to say defense does not hurt ranged,  but simply that they do much better against melee. Which means seiges tend to be primarily ranged units (for humans anyways). Allowing mangonels to throw cows would hurt ranged more than melee as melee is usually moving up to the wall, while the ranged is standing un place to fire (except ha)

**Comment by Russianstory** [comment]
I don’t remember a single case when the carcass of a dead cow was lifted up the tower and then shot. Well this is too much already. So unrealistic. I think this is an idea well, more for Legends. And it's so easy to leave everything as it is. For now.

**Comment by GRhin** [comment]
I am certain it would have happened. But you appear to be the historical expert. 
Instead you can imagine this as a standin for throwing explosive shells which definately did happen and should have a similar effect.
My suggestion was not a suggestion for historical accuracy (never even mentioned that) but for game balance. If you are looking for accuracy you wouldnt need to use it.

**Comment by GRhin** [comment]
The reality is, throwing corpses in history would not have been as effective as the attackers did not camp or seige the way you do in crusader, and you were not killed just by walking through the region the corpse landed, but often much mush later, already we see that the entire mechanic is not "realistic", but if it worked this way in real life, you can bet your ass that they would have done exactly this

**Comment by Russianstory** [comment]
Yes, I know a little more than ordinary schoolchildren know .. but there is also the Internet)) What you are offering is fantastic, much more abruptly than my suggestions for increasing the power of a longbow strike. Masha's suggestion to throw cows off the mangonels is a very bad idea. I do not know how this could come to mind.

**Comment by GRhin** [comment]
Power of a longbow is nerfed by shields, and counters melee more.
How could this come to mind? By reading my first post where I explained why I think this is a good idea. 
Reality is, building trebs behind the wall already achieves this much more effectively as they would be unable to be hit by arrow fire (protected by the wall), this way is cheaper, but the mangonels are more vulnerable to either arrow fire or the tower being destroyed.

**Comment by Halbarad1** [comment]
Stronghold isn't that correct, I think it could be pretty funny to allow mangonels to throw cows. 

![Monty Python Kuh](https://user-images.githubusercontent.com/44904062/95264268-4b9d0980-082f-11eb-9008-1ecc136325f9.gif)

**Comment by JuGGerNaunT** [comment]
![no-you-cant-just-](https://user-images.githubusercontent.com/1506510/95563775-87e68a80-0a26-11eb-85f6-e9ef2ed86546.png)
![ezgif com-optimize](https://user-images.githubusercontent.com/1506510/95564289-40acc980-0a27-11eb-9181-50c8969b6993.gif)

**Comment by GRhin** [comment]
That's a lot of cows lol

**Comment by Russianstory** [comment]
No ... it's strangely unrealistic and ugly ...

**Comment by GRhin** [comment]
I disagree with both points.
Regardless, as I said on a recent discord discussion, shc is not a 100% historically accurate game - in real life a dead cow doesnt kill you just from walking past it - therefore historical accuracy should not be the only arguement against a feature that is suggested to improve balance and gameplay, which is the reasoning I have given for this feature. 
I dont believe the feature is unrealistic in the least, and doesnt look any uglier than any other similar unit throwing cows (which they all do). And you are unable to give me a gameplay related

**Comment by Russianstory** [comment]
You're probably joking. Six COWS take off from the roof of the TOWER from ONE gun in one gulp! I have never seen a more ugly form of defense. It's disgusting. it's shit =). I am strongly opposed to this idea. But if you still want to do it, please do it as a separate function .... so that it is not necessary. For I don't like it.

**Comment by GRhin** [comment]
> That said, the image juggernaunt posted shows one mangonel throwing multiple cows, which is excessive, I am only intending to throw 1 at a time.
Kindly read my message before telling me how bad a feature is in a format that I am not proposing.
As for being in a separate function that is not needed. You are asking us to implement the feature in the same way that we have done every other feature we have introduced, why would you assume that this feature would be the one exception?

**Comment by Russianstory** [comment]
I see that the cow function you proposed is disgusting)) for the reasons I already mentioned). I will not repeat myself. Yes, please do just that, because if you cram "THIS IS" into the general patch, it will not be very good. Not everyone loves and wants to shoot six cows from a mangonel ...

**Comment by GRhin** [comment]
> Not everyone loves and wants to shoot six cows from a mangonel ...

correct, if you read either of my last two posts, you would see I dont want to do that either. Dont repeat yourself, read the response, and reply to that instead. That is a much more productive way of having a discussion.

**Comment by Lolasik011** [comment]
@GRhin I like the idea !

**Comment by Russianstory** [comment]
Are you also against mangonel cows?

**Comment by Lolasik011** [comment]
No, I'm for cows and mangonel. But only 1 cow per shot

**Comment by ilosevsrat** [comment]
1 cow per shot.  5 looks wierd AF IMHO =DD but, thats great stuff tho, WP Moder!

**Comment by JuGGerNaunT** [comment]
lol that obviously was a joke. i wanted to post it to reddit but thought it would be out of context there.
